tox.ini

```
@@ -306,6 +306,17 @@ deps =
     tomte[docs]==0.1.5
 commands = {toxinidir}/scripts/generate_api_docs.py --check-clean
 
+[testenv:generate-all-protocols]
+skipsdist = True
+usedevelop = True
+deps =
+    tomte[isort]==0.1.5
+    tomte[black]==0.1.5
+commands = pip install {toxinidir}[all]
+           pip install --no-deps file://{toxinidir}/plugins/aea-cli-ipfs
+           python -m aea.cli generate-all-protocols
+           python -m aea.cli generate-all-protocols tests/data/packages
```

Maybe we could refactor as well:

```suggestion
           aea generate-all-protocols
           aea generate-all-protocols tests/data/packages
```

---

good suggestion but I'm no rerunning the CI over it, plus two PRs stacked on top already

---

refactored